en/starter/generator/command_line_tools.md

```diff
@@ -0,0 +1,154 @@
+| **`yo swiftserver:property`**     | [Property generator](#property-generator)       | Add a new property to an existing model.                                                                                                                                                                                                                               |
+| **`yo swiftserver:refresh`**      | Refresh generator                              | Generate OpenAPI (Swagger 2.0) and product definition YAML files, by convention named *`application`*`.yaml` and *`application`*`-product.yaml`. If the product file already exists it is not regenerated. The OpenAPI (Swagger 2.0) definition is always regenerated. |
+
+---
+
+## Application generator
+
+**Synopsis**
+
+Creates a new Swift Server Generator application.
+
+    yo swiftserver [options] [name]
+
+**Options**
+
+    -h, --help
+
+Print the generator’s options and usage.
+
+    -n, --name
+
+Provide the application name.
+
+**Interactive prompts**
+
+The generator prompts you for the following information:
+
+-   The name of the application to create, if you have not provided the application name as an option, as in the following example:
+
+        yo swiftserver --name bookshop
+
+-   The name of the directory to create the application in; this defaults to the application name you previously entered.
+
+-   Data store to use; this defaults to memory which is an in-memory data source that is built into Swift Server Generator and is suitable only for development and initial testing. If you want to connect your models to a real data source such as the Cloudant® database server, choose `cloudant` for your back-end data store. The tool prompts you for additional settings and automatically installs the appropriate Cloudant connector package by using the Swift Package Manager. If you want to switch between data stores see the [`config.json file`](config_json.md).
+
+The tool creates the standard Swift Server Generator application structure; for details, see [Swift Server Generator project layout reference](project_layout_reference.md).
+
+After you create an application, you can run additional generators from the application root directory:
+
+-   [Model generator](#model-generator)
+
+-   [Property generator](#property-generator)
+
+---
+
+## Model generator
+
+**Important:** Before running this generator, you must create an application by using the [Application generator](#application-generator). Then you must run the command from the root directory of the application.
+
+**Synopsis**
+
+Adds a new model to a Swift Server Generator application:
+
+    yo create swiftserver:model [options] [name]
+
+**Options**
+
+Print the generator’s options and usage:
+
+    -h, --help
+
+Provide the model name:
+
+    -n, --name

```

Comment:
There is no `-n` or `--name` option. The name can be passed as an optional argument.